java/ql/src/utils/stub-generator/Stubs.qll

```diff
@@ -16,7 +16,12 @@ abstract private class GeneratedType extends ClassOrInterface {
   }
 
   private string stubKeyword() {
-    this instanceof Interface and result = "interface"
+    this instanceof Interface and
+    (
+      this instanceof AnnotationType and result = "@interface"
+      or
+      result = "interface"
```

Comment:
Doesn't this have to use `if ... else` because currently for `AnnotationType` both `@interface` and `interface` would be results?

Reply:
I could've sweared this worked just fine, but you're right that it looks it would behave as you say. Changing it, thanks.

```suggestion
    (if this instanceof AnnotationType then result = "@interface" else result = "interface")
```

Reply:
Done in 4f5763f.